src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -95,10 +106,22 @@ public void satisfiesEqualsContract() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            new RqMultipart.Base(req),
-            Matchers.equalTo(new RqMultipart.Base(req))
-        );
+        final RqMultipart.Base reqbaseone = new RqMultipart.Base(req);
+        final RqMultipart.Base reqbasetwo = new RqMultipart.Base(req);
+        try {
+            MatcherAssert.assertThat(
+                reqbaseone,
+                Matchers.equalTo(reqbasetwo)
+            );
+        } finally {
+            req.body().close();
+            reqbaseone.part("t-1")
+            .iterator()
+            .next().body().close();
```

Comment:
@raphaelln something is wrong with indentation here, please fill one line with as much as possible